### ISSUE TITLE
[REST API] : Page API returns publishDate and modDate fields with the same value

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#20791]
Reported via support: [104914](https://dotcms.zendesk.com/agent/tickets/104914)
Reproducible as of dotCMS 5.3.8.5, up to latest master (21.08)

The Page API returns a JSON representation of the internal components of an HTML Page. When such a page contains a Contentlet with a **Publish Date** field, the JSON response is explicitly assigning the `publishDate` attribute to have the date from the `modDate` property.

**To Reproduce**
1. Create a Content Type named `My Test CT` with two fields: a **Text** field named `Title`, and a **Date and Time** field named `Publish Date`
2. Create a Contentlet of type `My Test CT`.
3. Create or update a Container so that it can display such a type of content. And create a test HTML Page to add the Contentlet you created previously.
4. Use the Page API to request the page's data. For example: https://demo.dotcms.com/api/v1/page/render/test-page?language_id=1 , and scroll down to the description of the Contentlet. Notice how the `modDate` and `publishDate` fields have the exact same value:
```
contentlets: {
    uuid-1562770692396: [
        {
            hostName: "demo.dotcms.com",
            modDate: 1628869709464,
            publishDate: 1628869709464,
            title: "My Test Content",
            baseType: "CONTENT",
            ...
```

**Expected behavior**
The **Publish Date** field must be returned correctly and not overwritten with the `Mod Date` value, unless there's not **Publish Date** field defined in the Content Type.

**Comment by alfredo-dotcms** [comment]
fixed
![image](https://user-images.githubusercontent.com/37185433/131585498-66c0b00a-9a97-4896-abda-be73c814956b.png)

**Comment by bryanboza** [comment]
Fixed, tested on release 21.10 // docker // FF 
![image](https://user-images.githubusercontent.com/2641437/134229866-9f0e9d98-0552-4bdc-8436-dabfc5691287.png)
